backend/sirius-web-compatibility/src/main/java/org/eclipse/sirius/web/compat/forms/WidgetDescriptionConverter.java

```diff
@@ -105,6 +105,9 @@ private TextfieldDescription convertTextfield(org.eclipse.sirius.properties.Text
                 .labelProvider(labelProvider)
                 .valueProvider(valueProvider)
                 .newValueHandler(newValueHandler)
+                .diagnosticsProviders((variableManager) -> List.of())
```

Comment:
it should be `diagnosticsProvider` since there is one provider of potentially multiple diagnostics (this comment can be applied everywhere in this PR)

Reply:
Done